Update Kafka inbound endpoint setup guide (#180)
wso2/docs-mi: en/docs/reference/connectors/kafka-connector/setting-up-kafka.md

# Setting up Kafka

> ## For kafka inbound endpoint

Reviewer: ```suggestion
## For Kafka inbound endpoint
```

To configure the Kafka inbound endpoint, copy the following orbit bundles to the `<MI_HOME>/dropins` directory.

* [kafka-avro-serializer](https://mvnrepository.com/artifact/org.wso2.orbit.io.confluent/kafka-avro-serializer/7.6.0.wso2v1)
* [kafka-schema-serializer](https://mvnrepository.com/artifact/org.wso2.orbit.io.confluent/kafka-schema-serializer/7.6.0.wso2v1)
* [kafka-schema-registry-client](https://mvnrepository.com/artifact/org.wso2.orbit.io.confluent/kafka-schema-registry-client/7.6.0.wso2v1)

Copy the following additional client libraries to the `<MI_HOME>/lib` directory.

* [kafka-clients-3.7.0.jar](https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients/3.7.0)
* [avro-1.11.3.jar](https://mvnrepository.com/artifact/org.apache.avro/avro/1.11.3)

!!!Note

>     If you are using the Kafka connector as well, you will need to copy the necessary dependencies according to the following section.

Reviewer: ```suggestion
    If you are also using the Kafka connector, copy the necessary dependencies as outlined in the following section.
```
